python_modules/dagster/dagster/_utils/error.py

```diff
@@ -63,6 +66,16 @@ def to_exception_message_only(self) -> "SerializableErrorInfo":
         """
         return SerializableErrorInfo(message=self.message, stack=[], cls_name=self.cls_name)
 
+    @classmethod
+    def from_pipes_exc(cls, exc: "PipesException"):
+        return cls(
+            message=exc["message"],
+            stack=exc["stack"],
+            cls_name=exc["name"],
+            cause=cls.from_pipes_exc(exc["cause"]) if exc["cause"] else None,
+            context=cls.from_pipes_exc(exc["context"]) if exc["context"] else None,
+        )
+
```

Comment:
Minor thing, but I think this should be a function in pipes rather than making utils aware Pipes.